sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cas-integration/models/cas-supplier-response.dto.ts

```diff
@@ -0,0 +1,52 @@
+export class CASSupplierResponse {
+  items: CASSupplierResponseItem[];
+  hasMore: boolean;
+  limit: number;
+  offset: number;
+  count: number;
+  links: CASSupplierResponseItemLink[];
```

Comment:
In general, I would suggest to map only what the application will consume. For instance `links` will not be needed as far as I can see.

Reply:
Please also remove the `CASSupplierResponseItemLink` that no longer has references.